### PR TITLE
Import Bulk Invitations - Superadmin

### PR DESCRIPTION
**Pull request by Moiz1524**
## Description
This includes functionality to import bulk invitations through an excel file

### CODE REVIEW

File: Gemfile
Comment (sdnorm) [inline Gemfile:1]: Why was `devise_invitable` removed?

**Comment by sdnorm** [comment]
This PR will get merged along with https://github.com/SuperDupr/builder/pull/31. The discussion there directly applied to this PR as well.

File: Gemfile
Comment (Moiz1524) [reply Gemfile:1]: > Why was `devise_invitable` removed?

I have used Jumpstart default AccountInvitation setup. This is an alternative of devise_invitable gem. That's why no longer needed

**Comment by Moiz1524** [comment]
> This PR will get merged along with #31. The discussion there directly applied to this PR as well.

Understood

**Pull request merged**